packages/progress-bar/src/ProgressBar/ProgressBar.utils.tsx

```diff
@@ -112,14 +113,35 @@ export const getFormattedValue = (
   }
 };
 
-export const getValueAriaAttributes = (value?: number, maxValue?: number) => {
+export const getDivAttributes = (type: Type, label?: React.ReactNode) => {
```

Comment:
for this (and all other utils) can you add a quick TSDoc comment about what the util does?